packages/application-shell/src/components/project-switcher/project-switcher.tsx

```diff
@@ -67,85 +68,72 @@ export const ValueContainer = ({ ...restProps }: ValueContainerProps) => {
 
 export const ProjectSwitcherOption = (props: OptionProps) => {
```

Comment:
Maybe we should update the tests for this component.

Reply:
@CarlosCortizasCT which test case you had in mind for this component?

Reply:
Test that the projects listed in the dropdown render the expected labels based on their properties.

Reply:
@CarlosCortizasCT thanks! I added a test here - [965c3b2](https://github.com/commercetools/merchant-center-application-kit/pull/3403/commits/965c3b2a6ad28ae802311748f90d2d48105fb9f4). Let me know if you have any remarks.

Reply:
It seems the test is verifying the different stamps are rendered:
![image](https://github.com/commercetools/merchant-center-application-kit/assets/97907068/f654a870-b6d6-46f1-804f-700089dfb54b)

I think it would be great if we also test that a project row can also render several stamps at the same time.